app/src/components/App/Application/index.jsx

```diff
@@ -52,65 +53,30 @@ class Application extends Component {
     });
   }
 
-  async submitApplication() {
+  submitApplication() {
     const { name, email,
-      applicationText: application_text,
-      ordered: prioritized,
-      selectedComittees: committees,
+      applicationText,
+      ordered,
+      selectedComittees,
     } = this.state;
-    const application = {
-      name, email, prioritized, application_text,
-      committees: committees.map((committee, index) => Object.assign({
-        group: committeesMap.get(committee).id,
-        priority: index + 1,
-      })),
-    };
-
-    const uri = `${process.env.SG_APPLICATION_BACKEND}${process.env.SG_APPLICATION_ENDPOINT}`;
+  
+    const application = new CommitteeApplication(null, name, email, applicationText, selectedComittees, ordered);
```

Comment:
Since this is the only place CommitteeApplication is used I think the first parameter should be removed.